renderer/src/components/ActivityChartCard.jsx

```
@@ -210,31 +197,30 @@
-        <div className="flex justify-between">
+        <div className="flex justify-end space-x-2">
           <button 
             onClick={() => {
               setRetryCount(0);
               generateChart(true);
             }}
             disabled={loading}
-            className="px-3 py-1 bg-gray-200 text-gray-800 rounded hover:bg-gray-300 transition-colors disabled:opacity-50 flex items-center"
+            title="Refresh"
+            className="p-2 bg-gray-200 text-gray-800 rounded-full hover:bg-gray-300 transition disabled:opacity-50"
           >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)